_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,363 @@
+---
+id: ffbc54
+name: No keyboard shortcut uses only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable character keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][] for which all of the following is true: 
```

Comment:
I'm not too sure about targeting events with the applicability. It means this requires some additional "rigger the event" step to run this rule. That may not be done consistently, which would lead to inconsistent results. I think a better thing to do here is to apply this to event handlers instead.

Reply:
I am not sure triggering event handlers would solve this.
Testers do not necessarily have easy access to the event handlers (e.g. imported from some minified external lib). Plus, figuring out what the code actually does is undecidable…
So even with targeting handlers, testers may decide to trigger the event and see if changes happen.

Reply:
I agree with @Jym77 that moving to event handlers probably won't change the testing procedure. Furthermore, we had an initial version using event handlers, which proved to be more challenging to write (earlier comments can show you why).